charts/internal/machine-controller-manager/seed/templates/deployment.yaml

```diff
@@ -0,0 +1,110 @@
+    metadata:
+      annotations:
+        scheduler.alpha.kubernetes.io/critical-pod: ''
+{{- if .Values.podAnnotations }}
+{{ toYaml .Values.podAnnotations | indent 8 }}
+{{- end }}
+      labels:
+        gardener.cloud/role: controlplane
+        app: kubernetes
+        role: machine-controller-manager
+        networking.gardener.cloud/all: allowed
+{{- if .Values.podLabels }}
+{{ toYaml .Values.podLabels | indent 8 }}
+{{- end }}
+    spec:
+      serviceAccountName: machine-controller-manager
+      terminationGracePeriodSeconds: 5
+      containers:
+      - name: machine-controller-manager-provider-kubevirt
+        image: {{ index .Values.images "machine-controller-manager-provider-kubevirt" }}
+        imagePullPolicy: IfNotPresent
+        command:
+        - ./machine-controller
+        - --control-kubeconfig=inClusterConfig
+        - --target-kubeconfig=/var/lib/machine-controller-manager/kubeconfig
+        - --namespace={{ .Release.Namespace }}
+        - --port={{ .Values.metricsPort2 }}
+        - --v=3
+        livenessProbe:
+          failureThreshold: 3
+          httpGet:
+            path: /healthz
+            port: {{ .Values.metricsPort2 }}
+            scheme: HTTP
+          initialDelaySeconds: 30
+          periodSeconds: 10
+          successThreshold: 1
+          timeoutSeconds: 5
+        terminationMessagePath: /dev/termination-log
+        terminationMessagePolicy: File
+        volumeMounts:
+        - mountPath: /var/lib/machine-controller-manager
+          name: machine-controller-manager
+          readOnly: true
+      - name: machine-controller-manager
+        image: {{ index .Values.images "machine-controller-manager" }}
+        imagePullPolicy: IfNotPresent
+        command:
+        - ./machine-controller-manager
+        - --control-kubeconfig=inClusterConfig
+        - --target-kubeconfig=/var/lib/machine-controller-manager/kubeconfig
+        - --namespace={{ .Release.Namespace }}
+        - --port={{ .Values.metricsPort }}
+        - --machine-creation-timeout=20m
+        - --machine-drain-timeout=2h
+        - --machine-health-timeout=10m
+        - --machine-safety-apiserver-statuscheck-timeout=30s
+        - --machine-safety-apiserver-statuscheck-period=1m
+        - --machine-safety-orphan-vms-period=30m
+        - --machine-safety-overshooting-period=1m
+        - --safety-up=2
+        - --safety-down=1
+        - --v=3
+        livenessProbe:
+          failureThreshold: 3
+          httpGet:
+            path: /healthz
+            port: {{ .Values.metricsPort }}
+            scheme: HTTP
+          initialDelaySeconds: 30
+          periodSeconds: 10
+          successThreshold: 1
+          timeoutSeconds: 5
+        ports:
+        - name: metrics
+          containerPort: {{ .Values.metricsPort }}
+          protocol: TCP
+        resources:
+          requests:
+            cpu: 50m
+            memory: 64Mi
+          limits:
+            cpu: "3"
+            memory: 3000Mi
```

Comment:
Why are these values so large? On other providers, I find significantly lower values:
```yaml
          limits:
            cpu: 350m
            memory: 256Mi
```
Does the MCM provider sidecar require all these resources, and why?

Reply:
no reason, that was a copy paste from vsphere, let me check this

Reply:
i changed limits to suggested. done.